linalg/src/frame/reduce/mod.rs

```diff
@@ -280,7 +249,7 @@ pub mod test {
         let found = values;
         let red = op.run_with_params(found, params).unwrap();
         tensor0(red)
-            .close_enough(&tensor0(expected), true)
+            .close_enough(&tensor0(expected), Approximation::SuperApproximate)
```

Comment:
do we still need SuperApprox when we use integer-as-float inputs ?

Reply:
You're right, it should be fine with `Approximate`